datasets/flwr_datasets/visualization/comparison_label_distribution.py

```diff
@@ -243,3 +249,34 @@ def _initialize_comparison_xy_labels(
         xlabel, ylabel = ylabel, xlabel
 
     return xlabel, ylabel
+
+
+def _initialize_axis_sharing(
+    size_unit: Literal["percent", "absolute"],
+    plot_type: Literal["bar", "heatmap"],
+    partition_id_axis: Literal["x", "y"],
+) -> dict[str, bool]:
+    # Do not intervene when the size_unit is percent and plot_type is heatmap
+    if size_unit == "percent":
+        return {}
+    if plot_type == "heatmap":
+        return {}
+    if partition_id_axis == "x":
+        return {"sharey": True}
+    if partition_id_axis == "y":
+        return {"sharex": True}
+    return {"sharex": False, "sharey": False}
+
+
+def _set_tick_on_value_axes(
```

Comment:
would it make more sense to use the word `disable_tick` rather than `set_tick`?

Reply:
I can't google it. Could you send a link to matplotlib docs of `disable_tick`?

Reply:
oh, i mean the name of this function `_set_tick_on_value_axes`, but maybe i'm overthinking it